Assets/Scripts/UI/Menu's/MenuController.cs

```diff
@@ -1,93 +1,11 @@
 using UnityEngine;
 using UnityEngine.SceneManagement;
```

Comment:
Dit is natuurlijk de abstract class met deze 2 namespaces. Wat ik mijn nu af vraag, elke class die hiervan over evert, hebben die ook hun eigen namespaces nodig? Ik weet het niet zeker of het kan, dus aan jou om uit te zoeken. Als het antwoord is dat het weg kan, dan kan alle namespaces in de andere classes weg.